pages/_app.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Head from "next/head";
+
+function MyApp({ Component, pageProps }) {
+  return (
+    <>
+      <Head>
+        <link
```

Comment:
The library doesn't ship w/ the CSS, so the components won't look like anything by default. This link tag is added in a common _app.js file (https://nextjs.org/docs/advanced-features/custom-app) that will make sure the CSS is loaded on every page.